### ISSUE TITLE
Compatibility with CoreOS

### ISSUE THREAD

**Issue by AMarti96** [dcos/dcos-ansible#43]
As I can see in the official DC/OS documentation, CoreOS is one of the operating environments supported, but in this repo there is no compatibility for that, are you planning to add it in the future nearby?

My company is trying to deploy DC/OS on CoreOS machines and it will be great to have it managed via ansible.

**Comment by fatz** [comment]
You're right currently CoreOS is not supported by these ansible roles. There are technical issues which are not yet sorted out. So I cannot state any time when this might be available

**Comment by a-nldisr** [comment]
Since CoreOS is EOL, perhaps this can be closed?
